playbooks/provision-bastion/README.md

```diff
@@ -13,10 +13,13 @@ If the IdM / IPA integration is to be used, it is a prerequisites that the envir
 2. When enabling VNC, and you already have a shared home directory, make sure the proper changes are made to the VNC configuration (typically in `~/.vnc` ) to allow for the service to run correctly.
 
 ## Example run
-How to run the playbook may depend on the options selected. However, below is an example execution whereas the password for IPA/IdM integration (with `ipa_client_install` set to `True` in the inventory) is passed in rather than statically set in the inventory. Modify the inventory to your liking in `playbooks/bastion/inventory`, then at the top level of the repository, execute the following command:
+How to run the playbook may depend on the options selected. However, below is an example execution whereas the password for IPA/IdM integration (with `ipa_client_install` set to `True` in the inventory) is passed in rather than statically set in the inventory. Modify the inventory to your liking in `inventory/bastion/`, then at the top level of the repository, execute the following command:
 
 ```
-> ansible-playbook -i playbooks/bastion/inventory playbooks/bastion/install.yml -e 'ipa_password=<ipa/IdM password>'
+## provision bastion host
+> ansible-playbook -i playbooks/provision-bastion/inventory playbooks/bastion/bastion.yml
```

Comment:
the `bastion.yml` playbook already calls out to the `install.yml` playbook, so probably need to fix up this slightly ... maybe just change it to (note the newline between the two) :

```
## Provision, Install and Configure Bastion on a supported target hosting_infrastructure
> ansible-playbook -i playbooks/provision-bastion/inventory playbooks/bastion/bastion.yml -e 'ipa_password=<ipa/IdM password>'

## Install and configure bastion (no provisioning)
> ansible-playbook -i playbooks/provision-bastion/inventory playbooks/bastion/install.yml -e 'ipa_password=<ipa/IdM password>'
```

Reply:
update the README.md per your suggestions
I have noticed that the bastion.yml imports the install.yml as well just yesterday, thanks for pointing this out